Gemfile.rails_version

```diff
@@ -2,22 +2,19 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV['RAILS_VERSION'] || '5.2.6'
+env_rails_version = ENV['RAILS_VERSION'] || '6.0.0'
 
-ruby '~> 2.6.7'
+ruby '~> 3.0.0'
 
 gem 'rails', "~> #{env_rails_version}"
 
 case env_rails_version
 when "master"
   {github: "rails/rails"}
-when /^5.2/, /^5.1/, /^5.0/
+when /^6.1/, /^6.0/
   gem 'bootstrap-sass', '~> 3.3'
   gem 'jquery-rails'
   gem "sprockets-rails", "~> 3.2"
-
-  # A JS runtime is required for Rails 4.0+
-  gem 'therubyracer', '~> 0.12.3', platforms: :ruby
```

Comment:
this didn't seem like it was useful for the spec suite which is the only thing I really run so I just ripped it out.

it barely builds on intel macs and not arm macs at all

Reply:
I'm always down to remove things from this gem and ultimately want to remove the gem all together.